### PR TITLE
Update RealmSwift from 3.1.1 to 3.11

### PR DESCRIPTION
**Pull request by SaraSaber**
Update RealmSwift from 3.1.1 to 3.11.0 to support Xcode 10

### CODE REVIEW

File: Example/Example.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -376,7 +375,7 @@
 				TargetAttributes = {
 					1A42C2881C0E3882000F2137 = {
 						CreatedOnToolsVersion = 7.1.1;
-						DevelopmentTeam = 32F2T8EJ6G;
+						DevelopmentTeam = NJ65ARC6W7;
```
Comment (revolter) [inline Example/Example.xcodeproj/project.pbxproj]: You should discard these changes

**Review by hebertialmeida — CHANGES_REQUESTED** [review]
Hi @SaraSaber, thank you for your contribution!

Few things, the changes to `DEVELOPMENT_TEAM`, `DevelopmentTeam` pointed out by @revolter as well `PRODUCT_BUNDLE_IDENTIFIER` are not necessary.

Could you update that?

**Comment by kdvmgn** [comment]
Are all issues solved? Could you merge this pull request, please?)

**Comment by hebertialmeida** [comment]
There are many unrelated changes on this PR, I will close it for now as this should be just a podfile change.

**Pull request closed without merging**